start.js

```diff
@@ -1,2 +1,2 @@
-console.log('Hello!');
-console.log('World');
\ No newline at end of file
+console.log('Hello!!!');
```

Comment:
dziwna zmaina nie wiem o co chodzi, mógłbyś lepiej opisywać kod?

Reply:
chodziło mi o to żeby wszyscy usłyszeli jak się witam stąd trzy wykrzykniki

Reply:
ah trzeba było tak od razu. co powiesz na tą sugestię:
```suggestion
console.log('Hello wszystkim!!!!!!');
```

Reply:
Wygląda sensownie. Może przerzucimy się na jeden język? Proponuję macedoński
```suggestion
console.log('Здраво на сите');
```